runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -379,4 +414,4 @@ private static String buildToolTip(LootTrackerItem item)
 			+ "<br>GE: " + QuantityFormatter.quantityToStackSize(gePrice)
 			+ "<br>HA: " + QuantityFormatter.quantityToStackSize(haPrice) + "</html>";
 	}
-}
+}
```

Comment:
fix your IDE to not always put a newline at the end of files, in intellij the option is `ensure link break at end of file on save` or something like that

Reply:
Done, thank you.